lib/msal-common/src/authority/AuthorityOptions.ts

```diff
@@ -12,4 +12,22 @@ export type AuthorityOptions = {
     cloudDiscoveryMetadata: string;
     authorityMetadata: string;
     azureRegionConfiguration?: AzureRegionConfiguration;
+    azureCloudInstance?: AzureCloudInstance;
 };
+
+export enum AzureCloudInstance {
```

Comment:
I don't see this enum exported from Node or Browser. I assume the intent is that customers would use this enum? If so, please reexport (otherwise this doesn't need to be a minor bump).

Reply:
Yes. Let me add this.

Reply:
Addressed.